core/services/blockhashstore/common.go

```diff
@@ -134,3 +139,15 @@ func SendingKeys(fromAddresses []ethkey.EIP55Address) []common.Address {
 	}
 	return keys
 }
+
+func RegisterLogPoller(coordinator Coordinator, logPoller logpoller.LogPoller, filterName string) error {
```

Comment:
```suggestion
func RegisterLogPoller(logPoller logpoller.LogPoller, filterName string, addresses ...common.Address) error {
```

Reply:
The more I think on this the more I feel like we should follow the Go proverb of [A little copying is better than a little dependency.](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s). Explicit log poller register filter calls within each delegate seems cleaner to me than having it abstracted away into this function.

Reply:
Jin also made this suggestion, I've broken this filter up to be in each coordinator's initializer.